README.md

```diff
@@ -19,12 +19,8 @@ package feed. This allows source-specific routes being transported properly.
 The build can make use of an apt cache.
```

Comment:
is this still accurate?

Reply:
It is not, thank you for pointing that out.

I have also remove the reference to a specific babeld version.